### PR TITLE
Fix/cancel render

### PR DESCRIPTION
**Pull request by akivab**
Allow for cancelling rendering

### CODE REVIEW

File: modules/tracktion_engine/model/export/tracktion_Renderer.cpp [hunk 1]
```diff
@@ -446,8 +447,16 @@ bool Renderer::renderToFile (const juce::String& taskDescription,
             }
             else
             {
-                while (task->runJob() == juce::ThreadPoolJob::jobNeedsRunningAgain)
-                {}
+                while (true) {
+                    if (cancelRenderTask && *cancelRenderTask) {
+                        DBG("Cancelling rendering task!!!");
+                        turnOffAllPlugins (edit);
```
Comment (jklein24) [inline modules/tracktion_engine/model/export/tracktion_Renderer.cpp:453]: Why is this here for cancelling? Could that cause issues if there's another render happening?

Comment (akivab) [reply modules/tracktion_engine/model/export/tracktion_Renderer.cpp:453]: are you asking re: turnoffallplugins? was doing it here because it was done in the case of successful render and thought that returns to same state in either case

Comment (jklein24) [reply modules/tracktion_engine/model/export/tracktion_Renderer.cpp:453]: Yeah for turnOffAllPlugins. Maybe it's fine, but was slightly confused by it when canceling rendering.

**Review by jklein24 — APPROVED** [review]
